core/common/src/main/java/alluxio/conf/InstancedConfiguration.java

```diff
@@ -509,6 +509,13 @@ private void checkTieredLocality() {
     }
   }
 
+  @Override
+  public int hashCode() {
```

Comment:
if we implement `hashCode` we also need to implement `equals`

Reply:
still working on it